ui/component/claimPreviewSubtitle/view.jsx

```diff
@@ -34,9 +42,18 @@ function ClaimPreviewSubtitle(props: Props) {
           <UriIndicator uri={uri} link />{' '}
           {!pending && claim && (
             <>
-              {isChannel &&
-                type !== 'inline' &&
-                `${claimsInChannel} ${claimsInChannel === 1 ? __('upload') : __('uploads')}`}
+              {isChannel && type !== 'inline' && (
+                <>
+                  <br />
+                  <span className="claim-preview-sub-count">
+                    {formattedSubCount} {subCount !== 1 ? __('Followers') : __('Follower')}
+                  </span>
+                  &nbsp;&bull;
```

Comment:
@mayeaux, thoughts?  I think spacing should be controlled by css, or at the very least use `{' '}` instead for readability.